python/letsql/backends/let/hotfix.py

```diff
@@ -126,7 +127,12 @@ def exists(self):
 def _letsql_find_backend(self, *, use_default=True):
     # FIXME: push this into LETSQLAccessor
     try:
-        current_backend = self._find_backend._original(self, use_default=use_default)
+        if tuple(op.source for op in self.op().find(Read)):
+            current_backend = letsql.options.backend
```

Comment:
I think `letsql.options.backend` may be null if it has not been initialized; in general, it is safer to do:
```python
    from letsql.config import _backend_init
    current_backend = _backend_init() 
```